### DISCUSSION TITLE
Wrong torrent name when downloading with wget

### DISCUSSION THREAD

**Discussion by AyaanZaveri** [KiraLT/torrent-stream-server#255] [Q&A] [answered]
Whenever I download a file, it ends up being magnet... instead of the actual file name.

**Comment by KiraLT** [comment, marked as answer]
`wget` does not respect `Content-Disposition` header which indicates filename, so you need to add `--content-disposition` option:

```bash
wget --content-disposition 'http://localhost:3000/stream/08ada5a7a6183aae1e09d831df6748d566095a10'
```

The equivalent thing using `curl`:

```bash
curl -J -O 'http://localhost:3000/stream/08ada5a7a6183aae1e09d831df6748d566095a10'
```

Source: [unix.stackexchange.com](https://unix.stackexchange.com/a/394467)
